pkg/volume/flexvolume/driver-call.go

```diff
@@ -51,6 +50,7 @@ const (
 	optionReadWrite = "kubernetes.io/readwrite"
 	optionKeySecret = "kubernetes.io/secret"
 	optionFSGroup   = "kubernetes.io/fsGroup"
+	optionMountsDir = "kubernetes.io/mountsDir"
```

Comment:
Also add "securityContext" to options. I realized in testing we need it for SELinux to work seemlessly when the mount & actual mount path is handled by plugin.